### DISCUSSION TITLE
Access restriction on Elsa UI

### DISCUSSION THREAD

**Discussion by Rummy99** [elsa-workflows/elsa-core#3089] [Q&A]
Is it possible to Elsa UI with read only permissions. I need to have the UI with only workflow instances showing in the menu with delete and cancel options disabled.

**Comment by sfmskywalker** [comment]
There's no feature for this, but I am collecting functional requirements for Elsa 3's designer so please feel free to open an issue requesting this feature. Thanks!

**Comment by mohdali** [comment]
This is definitely required on designer and API level. Probably Policy based authorization needs to be integrated and then the user can plug their own policy handlers that can allow RBAC on workflow level. For the designer it needs to check if the user can execute a certain action.
